File: editorial_ch/chap2-5/prob2-5-3.java
import java.util.*;

class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		// 輸入
		long N = sc.nextInt();

		// 計算答案
		long Answer = 1;
		for (int i = 1; i <= N; i++) Answer *= i;

		// 輸出
		System.out.println(Answer);
	}
};

File: editorial_ch/chap2-5/prob2-5-3.cpp
#include <iostream>
using namespace std;

int main() {
	long long N;
	long long Answer = 1;
	cin >> N;
	for (int i = 2; i <= N; i++) Answer *= i; // 將Answer 乘以 i
	cout << Answer << endl;
	return 0;
}